mm2src/coins/z_coin.rs

```diff
@@ -734,6 +737,32 @@ pub async fn z_coin_from_conf_and_params(
     builder.build().await
 }
 
+fn verify_checksum_zcash_params(
+    spend_path: PathBuf,
```

Comment:
You can use `&PathBuf` here to avoid cloning.

Reply:
done